compiler/plugins/target/AMD-AIE/air/CMakeLists.txt

```diff
@@ -61,6 +61,18 @@ iree_tablegen_library(
     -gen-op-interface-defs Dialect/AIR/AIROpInterfaces.cpp.inc
 )
 
+iree_cc_library(
+  NAME
+    AIRTransformOps
```

Comment:
This is needed when registering the AIRTransformPass source files, because some of which contains transform.air ops.